File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.py
n = int(input())

while(n > 0):
    arr = input()
    total, seq = 0, 1
    for i in range(len(arr)):
        if(arr[i] == 'O'):
            if(i > 0 and arr[i - 1] == 'O'): seq += 1
            else: seq = 1
            total += seq * 1
    print(total)
            
    n -= 1

File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.c
#include <stdio.h>

int main(void) {
	int n;
	char arr[80];
	scanf("%d", &n);
	while (n--) {
		scanf("%s", arr);
		int total = 0;
		int seq = 1;
		for (int i = 0; i < strlen(arr); i++) {
			if (arr[i] == 'O') {
				if (i > 0 && arr[i - 1] == 'O') seq++;
				else seq = 1;
				total += 1 * seq;
			}
		}
		printf("%d\n", total);
	}

	return 0;
}